web/src/realm/RealmApolloProvider.tsx

```diff
@@ -1,38 +1,57 @@
 import * as React from "react";
-import * as RealmWeb from "realm-web";
+import * as Realm from "realm-web";
 import { useRealmApp } from "./RealmApp";
 
 // Apollo
-import { ApolloClient, HttpLink, InMemoryCache } from "apollo-boost";
-import { setContext } from "apollo-link-context";
-import { ApolloProvider } from "@apollo/react-hooks";
+import {
+  ApolloProvider,
+  ApolloClient,
+  HttpLink,
+  InMemoryCache,
+} from "@apollo/client";
+import { NormalizedCacheObject } from "@apollo/client/cache";
 
 const RealmApolloProvider: React.FC = ({ children }) => {
   const { id, user } = useRealmApp();
   const [client, setClient] = React.useState(
-    createApolloClient(id, user as RealmWeb.User)
+    createApolloClient(id, user as Realm.User)
   );
   React.useEffect(() => {
-    setClient(createApolloClient(id, user as RealmWeb.User));
+    setClient(createApolloClient(id, user as Realm.User));
   }, [id, user]);
 
   return <ApolloProvider client={client}>{children}</ApolloProvider>;
 };
 export default RealmApolloProvider;
 
 // TODO: Implement createApolloClient()
-function createApolloClient(realmAppId: string, user: RealmWeb.User) {
+function createApolloClient(
```

Comment:
Question: Is the comment on the line above still valid?

Reply:
Yup the `main` branch is the "complete" version of the `todo` branch, so this reflects that. I could see this potentially being confusing but haven't heard any issues yet.